client/src/styles/ButtonForChats.jsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+  export const ButtonForChats = styled.button`
+    border-radius: 0.6rem;
+    border: none;
+    width: 34rem;
+    height: 5rem;
+    padding: 1rem 0.5rem 1rem 0.5rem;
+    cursor: pointer;
+    background-color: ${({bg}) => bg || "#50924E"};
+    color: #fff;
+    font-style: italic;
+    transition: background-color 0.3s;
+    position: fixed;
+    left: 50%;
+    transform: translateX(-50%);
+    bottom: 4rem;
+    z-index: 2;
+
+    &:hover{
+      background-color: ${({hoverBg}) => hoverBg || "#396d37"};
+    };`
+const StyledButton = ({ to, text, ...rest }) => {
```

Comment:
DRY - this button is the same as the button in [client/src/styles/ButtonConversation.jsx](https://github.com/Makes-Innovation-Hub/arabrew/pull/54/files#diff-ebf05d9035553de695c094c8462eb148b65df83190c4a7cff4906460f33480a9)

Reply:
The problem is his position- in other states the position is different so I did other styled-components for this

Reply:
מדוייק יותר-
the different are- ButtonForChats as to be in the same position-no matter how much content the page contains(with position fixed and all his stuff)- while ButtonConversation depends on text (by changes with a screen size the position of the button will change)